### PR TITLE
Added host and database_name to Kubernetes secret

### PR DESCRIPTION
**Pull request by bartlemstra**
I need the host and database_name separately in my application, so I added them to the secret.

### CODE REVIEW

**Comment by arnarg** [comment]
Are you unable to use the `POSTGRES_URL` in the form of `postgresql://<user>:<pass>@<host>/<database>`?

**Comment by bartlemstra** [comment]
That doesn't seem to work: `Caused by: java.lang.RuntimeException: Driver org.postgresql.Driver claims to not accept jdbcUrl, postgresql://***`

**Comment by arnarg** [comment]
Can you prepend `jdbc:` in front of it? So it's `jdbc:postgresql://<user>:<pass>@<host>/<database>`.

https://jdbc.postgresql.org/documentation/80/connect.html

**Comment by bartlemstra** [comment]
Thanks for the suggestion, but that doesn't help:
`Caused by: java.lang.RuntimeException: Driver org.postgresql.Driver claims to not accept jdbcUrl, jdbc:postgresql://***`

File: pkg/controller/postgresuser/postgresuser_controller.go [hunk 1]
```diff
@@ -273,10 +273,12 @@ func (r *ReconcilePostgresUser) newSecretForCR(cr *dbv1alpha1.PostgresUser, role
 			Labels:    labels,
 		},
 		Data: map[string][]byte{
-			"POSTGRES_URL": []byte(pgUserUrl),
-			"ROLE":         []byte(role),
-			"PASSWORD":     []byte(password),
-			"LOGIN":        []byte(login),
+			"POSTGRES_URL": 	[]byte(pgUserUrl),
+			"HOST":				[]byte(r.pgHost),
```
Comment (arnarg) [inline pkg/controller/postgresuser/postgresuser_controller.go:277]: Fix indentation

**Comment by hitman99** [comment]
@arnarg looks good

**Comment by hitman99** [comment]
postgres-operator 0.4.3 was released with these changes: https://github.com/movetokube/postgres-operator/releases/tag/0.4.3

**Pull request merged**